### PR TITLE
Feedback from support week 26

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

This work follows feedback from our support team and from the accessibility audit.

## Proposal

- Add a new placeholder "catch phrase" on the course,
- Extend the plain text and simple text plugins to allow configuring their max length and CKEditor on specifc placeholders,
- Configure max length and CKEditor on course, person and blog post placeholders, 
- solve padding issues on the course licence,
- change the person bio to a plain text plugin,
- update README to the latest situation
- fix empty page on the annex link at the bottom of the sitemap

### CODE REVIEW

File: src/frontend/scss/templates/courses/cms/_course_detail.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@ $richie-course-detail-title-underline-border: 1px solid $gray40 !default;
 
 $richie-course-detail-content-width-large: 70% !default;
 $richie-course-detail-content-margin: 0 !default;
-$richie-course-detail-content-padding: 0 !default;
+$richie-course-detail-content-padding: 0 0 30px 0 !default;
```
Comment (mbenadda) [inline src/frontend/scss/templates/courses/cms/_course_detail.scss]: You should use `rem`

File: src/frontend/scss/templates/courses/plugins/_licence_plugin.scss [hunk 2]
```diff
@@ -20,8 +20,13 @@ $richie-licence-plugin-description-fontcolor: $gray40 !default;
   }
 
   &__wrapper {
+    margin-left: 10px;
```
Comment (mbenadda) [inline src/frontend/scss/templates/courses/plugins/_licence_plugin.scss]: Same here, better to use `rem`

File: src/frontend/scss/templates/courses/plugins/_licence_plugin.scss [hunk 3]
```diff
@@ -20,8 +20,13 @@ $richie-licence-plugin-description-fontcolor: $gray40 !default;
...
     @include media-breakpoint-up($richie-licence-plugin-breakpoint) {
-      @include sv-flex(1, 0, calc(100% - #{$richie-licence-plugin-logo-width}));
+      @include sv-flex(
+        1,
+        0,
+        calc(100% - #{$richie-licence-plugin-logo-width + 10})
```
Comment (mbenadda) [inline src/frontend/scss/templates/courses/plugins/_licence_plugin.scss]: This `10` comes from the margin above right? It should then be set in a variable and reused

Comment (sampaccoud) [reply src/frontend/scss/templates/courses/plugins/_licence_plugin.scss]: @mbenadda my problem is that the logo width is expressed in px so I get the error:

> "Incompatible units: 'rem' and 'px'."

Comment (sampaccoud) [reply src/frontend/scss/templates/courses/plugins/_licence_plugin.scss]: So I left px as discussed.

**Pull request merged**